### PR TITLE
Basic node interface

### PR DESCRIPTION
**Pull request by CDuPlooy**
Basic node interface as requested by @FJMentz

### CODE REVIEW

File: src/app/services/node/node.service.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/**
+ * Handles integration of the networking components present in docker.
```
Comment (egeldenhuys) [inline src/app/services/node/node.service.ts]: Is this the correct description?

File: src/app/models/node/node.model.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  Index: number;
+}
+export interface Spec {
+  Name: string;
+  Role: string;
+  Availability: string;
+  Labels: Labels;
+}
+export interface Labels {
+  foo: string;
```
Comment (egeldenhuys) [inline src/app/models/node/node.model.ts]: I think the `foo` is only an example

File: src/app/services/node/node.service.ts (hunk 1 above)
Comment (CDuPlooy) [reply src/app/services/node/node.service.ts]: I totally did not blatantly copy paste from another service.

**Review by egeldenhuys — APPROVED** [review]

**Review by FJMentz — APPROVED** [review]
Looks good! I will create a page (listview) for the nodes in this week!

**Pull request merged**